File: 백준/Bronze/17608. 막대기/막대기.py
import sys
input = sys.stdin.readline

stick = []
n = int(input())
for i in range(n):
    stick.append(int(input()))

height = 0
ans = 0
for i in range(n-1, -1, -1):
    if stick[i] > height:
        ans += 1
        height = stick[i]

print(ans)

File: 백준/Bronze/17608. 막대기/막대기.cc
#include <iostream>
#include <vector>

using namespace std;

int main()
{
	int n, tmp;
	cin >> n;

	vector<int> stick;
	for (int i = 0; i < n; i++) {
		cin >> tmp;
		stick.push_back(tmp);
	}

	int height = 0;
	int ans = 0;
	for (int i = n; i >= 0; i--) {
		if (stick[i] > height) {
			ans++;
			height = stick[i];
		}
	}

	cout << ans << endl;
	return 0;
}